frontend/js/src/entity-pages/AlbumPage.tsx

```diff
@@ -164,13 +168,16 @@ export default function AlbumPage(props: AlbumPageProps): JSX.Element {
           (track): Listen =>
             popularRecordingToListen(
               merge(track, {
-                artist_name: artistName,
-                artist_mbids:
-                  artist?.artists?.map((ar) => ar.artist_mbid) ?? [],
+                artist_name: track.artists
+                  .map(
+                    (track_artist) =>
+                      `${track_artist.artist_credit_name}${track_artist.join_phrase}`
+                  )
+                  .join(""),
```

Comment:
Let's do this for now; in the future I think we can refactor the [`getArtistName` utility](https://github.com/metabrainz/listenbrainz-server/blob/master/frontend/js/src/utils/utils.tsx#L231) to accept more input formats.

We probably want to keep the artist_mbids in there, as they can be used for other purposes down the line such as CB reviews

Reply:
Sounds good.